.github/workflows/main.yaml

```diff
@@ -37,12 +39,51 @@ jobs:
         with:
           node-version: 18.x
           cache: "yarn"
+  
+      - name: "Install shared module dependencies: vaa-core"
+        run: yarn workspace vaa-core install --frozen-lockfile
 
-      - name: "Install shared module dependencies"
-        run: yarn workspace vaa-shared install --frozen-lockfile
+      - name: "Build shared module: vaa-core"
+        run: yarn workspace vaa-core build
 
-      - name: "Build shared module"
-        run: yarn workspace vaa-shared build
+      - name: "Install shared module dependencies: vaa-data"
```

Comment:
The tests for the shared modules are combined in the same job because we need to anyway install and build all of them for the frontend.